sonar-db/src/main/java/org/sonar/db/rule/RuleDao.java

```diff
@@ -58,15 +55,15 @@ public RuleDto selectOrFailByKey(DbSession session, RuleKey key) {
   }
 
   public List<RuleDto> selectByIds(DbSession session, List<Integer> ids) {
-    return executeLargeInputs(ids, new IdToDto(mapper(session)));
+    return executeLargeInputs(ids, partitionOfIds -> mapper(session).selectByIds(partitionOfIds));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1612)